### ISSUE TITLE
[Feature Request] Add option to File Menu to copy path to file to the clipboard

### ISSUE THREAD

**Issue by netham45** [notepad-plus-plus/notepad-plus-plus#5870]
It would be convenient if there were options under the File menu to copy some file information to the clipboard. Something like:

"c:\test\file.txt"
File ->
... File Info->
... ... Copy File Name to Clipboard - Copies "file.txt"
... ... Copy Full Path to Clipboard - Copies "c:\test\file.txt"
... ... Copy File Location to Clipboard - Copies "c:\test\"

**Comment by sasumner** [comment]
Right-click a file's tab and you will find all that capability there.  Also on the Edit menu under "Copy to Clipboard".  Please close this issue.

**Comment by ArkadiuszMichalski** [comment]
@sasumner Maybe close if it already exists?

**Comment by GraphiteWingedBird** [comment]
Question about this - is there anyway to change what happens when you right-click on a file's tab? I can't find any type of Context Menu for what is available there.

The real reason I'm searching GitHub though and happened upon this old issue is that I am trying to find a way to change how the Copy to Clipboard work. My personal quest is to add double quotes to `Full File Path` and `Current Dir. Path` whenever they're copied to the clipboard, but depending on where the actual Copy to Clipboard code is nested/implemented, having some kind of configurability there would be awesome. E.g. Feature Request https://github.com/notepad-plus-plus/notepad-plus-plus/issues/10433 folks want the option to use forward slashes instead of backslashes. It seems like there is a ton of work happening or maybe completed, https://github.com/notepad-plus-plus/notepad-plus-plus/pull/9403, to make that happen on a grand scale - but what about just configurability of the `Copy to Clipboard` actions themselves?

In summary - is there anyway whatsoever that I can either default in double quotes when I copy a File Path or can I add my own actions to the File Tab Context Menu? Or since there is a Pop-Up Context Menu, do you know of a way/plugin to create new actions with names like `Copy Full File Path with Quotes`?
